lang/rust/Makefile

```diff
@@ -73,6 +73,12 @@ HOST_CONFIGURE_ARGS = \
 	--set=llvm.download-ci-llvm=true \
 	$(TARGET_CONFIGURE_ARGS)
 
+define Host/Prepare
```

Comment:
please use tar options instead:
https://github.com/openwrt/packages/blob/7709b85f86f7ce2da5a86f2d5d1fcdd77c5089a1/net/librespeed-go/Makefile#L28-L29

Reply:
I pushed a fix for this yesterday.